app/lib/service/announcement/backend.dart

```diff
@@ -21,7 +23,10 @@ class AnnouncementBackend {
 
   /// Loads the active announcement from Datastore.
   Future<void> update() async {
-    // TODO: implement loading announcements from Datastore.
+    final value = await secretBackend.lookup(SecretKey.announcement);
+    if (value != null && value.trim().isNotEmpty) {
+      _announcementHtml = value.trim();
```

Comment:
Can we run this through sanitize_html? I assume it's going to use `<b>` and `</a>` and maybe `<br>`... Just thinking it's better to have some defense in depth, whenever we inject HTML, if we can sanitize it we should.